man/systemd.link.xml

```diff
@@ -548,6 +548,15 @@
           cleared.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>WakeOnLanSecureOnPassword=</varname></term>
+        <listitem>
+          <para>Specifies the SecureOn(tm) password for MagicPacket(tm). Takes 6 bytes in Ethernet MAC
+          hex format (<literal>aa:bb:cc:dd:ee:ff</literal>). This will be ignored when
+          <varname>WakeOnLan=</varname> is specified without <literal>secureon</literal>. Defaults to
+          unset, and the current value will not be changed.</para>
+        </listitem>
+      </varlistentry>
```

Comment:
I think we should also make sure that if someone sets WakeOnLanPassword= but doesn't touch WakeOnLan= at all that we imply WakeOnLan=secureon.